build/three.js

```diff
@@ -18599,6 +18599,7 @@
 
 			if (capabilities.isWebGL2) {
 				// Workaround for https://bugs.chromium.org/p/chromium/issues/detail?id=1120100
+				// Not needed in Chrome 93+
```

Comment:
Do you mind removing the build files from your PR?

Reply:
@Mugen87 done